core/embed/rust/src/trezorhal/bitblt.rs

```diff
@@ -0,0 +1,216 @@
+use super::ffi;
+
+use crate::ui::{
+    display::Color,
+    geometry::Rect,
+    shape::{Bitmap, BitmapFormat, BitmapView},
+};
```

Comment:
After refactoring `bitblt.rs`, it would be impossible to split its implementation as you proposed. Therefore, I just added the enabling of the module with the `ui` feature as we already use the same approach with the display module, see d2ffa287ccda1d2b1a93888ecb479fc628b5b473